docs/reference/slate/schema.md

```diff
@@ -303,17 +303,34 @@ When supplying your own `normalize` property for a schema rule, it will be calle
 
 Raised when the `object` property of a child node is invalid.
 
-### `'child_required'`
+### `child_required_underflow`
```

Comment:
Could we change these to be `child_min_invalid` and `child_max_invalid` instead? That way they maintain the parallel with their definitions.